injector/LogInjector.py

```
@@ -0,0 +1,158 @@
+        self.processTree(rootNode.body, node.astNode.body)
+        return sstRootNode
+
+    def processChildNode(self, childNode, injectedTree):
+        '''
+            Given a child AST node, this function adds the node into
+            sst. It then inserts the node and the corresponding log 
+            statements into the injected tree. If the node is of type 
+            import, it is processed to check for locally imported files.
+        '''
+        node = NodeExtractor(childNode)
+        sstChildNode = self.sst.addAstNode("child", node, False)
```

Ruff (0.8.2)

Local variable `sstChildNode` is assigned to but never used

Remove assignment to unused variable `sstChildNode`

(F841)